app/controllers/stories_controller.rb

```diff
@@ -126,7 +126,26 @@ def handle_base_index
     @stories = ArticleDecorator.decorate_collection(@stories)
     set_surrogate_key_header "main_app_home_page"
```

Comment:
as this is edge cached and we're using the same surrogate key for both representations (HTML and JSON) I'm not 100% sure we can do that (ie. if Fastly honors the `Content-Type` in serving the edge response). 

See <https://docs.fastly.com/en/guides/getting-started-with-surrogate-keys> which is not clear about that.

I think by having the same URL and the same surrogate key, the first to be cached (either HTML or JSON) will be stored in the edge, but at the same time I would imagine the CDN to be smarter than that and respect the "Accept" header. Unfortunately I can't find that information, so we might have to test it but maybe we shouldn't on the homepage endpoint :D

Reply:
I'll follow @benhalpern's suggest to pop this into a new method that won't be cached (or can be cached seperately)